src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/repository/PrisonAppointmentRepository.kt

```diff
@@ -33,6 +33,15 @@ interface PrisonAppointmentRepository : JpaRepository<PrisonAppointment, Long> {
     endTime: LocalTime,
   ): Boolean
 
+  @Query(
+    value = """
+    SELECT pa FROM PrisonAppointment pa 
+     WHERE pa.videoBooking.statusCode = 'ACTIVE'
+       AND pa.appointmentDate = :date
+  """,
+  )
+  fun findAllActivePrisonAppointmentsOnDate(date: LocalDate): List<PrisonAppointment>
```

Comment:
We could further filter this on appointment type perhaps e.g. VLB_COURT_MAIN/VLB_PROBATION?